fairlearn/metrics/_make_derived_metric.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+import functools
+import inspect
+import numpy as np
+from typing import Callable, List, Union
+
+from ._metric_frame import MetricFrame
+
+transform_options = [
+    'difference',
+    'group_min',
+    'group_max',
+    'ratio',
+]
+
+_METRIC_CALLABLE_ERROR = "Supplied metric object must be callable"
+_METHOD_ARG_ERROR = "Callables which accept a 'method' argument " \
+    "may not be passed to make_derived_metric()"
```

Comment:
I'd include something here pointing users to `partial()` for them to know how to solve the issue.

Reply:
Good point. Will update.